### ISSUE TITLE
Crosshair stutters and shakes (version 1.19.2, quilt loader, dynamic crosshair + dynamic crosshair compat)

### ISSUE THREAD

**Issue by Duranson1** [Crendgrim/DynamicCrosshair#72]
https://github.com/Crendgrim/DynamicCrosshair/assets/88769753/261429bb-a7af-42d3-af03-32a70aae2657

As shown in the attached video. I am running quite a few mods but I've disabled the ones that I thought might be game breaking. Exordium is not installed. If a full mod list is needed, please let me know.

**Comment by Crendgrim** [comment]
It also appears to be significantly offset to the left. I've never seen this before. If you could limit it to some smaller set of mods, that would be greatly helpful. Otherwise, please do provide the list.

One guess I have is that *maybe* Raised is involved? But if so, it would need another mod interaction. Very intriguing.

**Comment by Duranson1** [comment]
After further testing, issue does seem to be with Raised and only raised.. versions 3.0.0-3.1.0 are incompatible, but 2.0.0 works just fine and is stutter-free. Looks like 3.1.0 completely rewrote the mod and evidently did something to cause an incompatibility with DC.

**Comment by Crendgrim** [comment]
I thought I tested that, hm. Probably getting confused with my other mods. Thank you, will investigate!

**Comment by Crendgrim** [comment]
Running Minecraft 1.19.2, DynamicCrosshair 5.7.1, Raised 3.1.0 (+ClothConfig & ModMenu) seems fine. I cannot reproduce your issue at all :/

Did you still have other mods installed when you could last replicate it?

**Comment by Duranson1** [comment]
Yeah, had quite a few mods installed. So I guess it must be raised+DC and some other mod causing the issue. I will figure out what the other incompatible mod is and get back to you.

**Comment by Duranson1** [comment]
Oh my god.. after all of that, turns out I had Exordium installed alongside everything.. I was so sure I removed it 😭anyways, working just fine now with Raised 3.1.0 and DC, sorry about that!!

**Comment by Crendgrim** [comment]
Hah. Glad you figured it out!

Last time I looked at it, Exordium worked with DC when turning off `enableGui`. Not sure how much it still helps, then, though.
